README.md

```diff
@@ -39,35 +44,16 @@ to clarify expected behavior in our community. For more information, see the [.N
 Please contribute. We are primarily interested in the following:
 
 * Improving quality and capability of the drivers for supported boards.
-* Implementations for additional boards
+* Implementations for additional boards.
 * [.NET device bindings](src/devices) for a wide variety of sensors, chips, displays and other components.
-* Links to blog posts or tweets that showcase .NET Core being used for great IoT scenarios (file an issue).
-* Request a device binding or protocol that you need for your project (file an issue).
+* Request a device binding or protocol that you need for your project ([file an issue](https://github.com/dotnet/iot/issues)).
+* Links to blog posts or tweets that showcase .NET Core being used for great IoT scenarios ([file an issue](https://github.com/dotnet/iot/issues)).
 
 ## Roadmap
 
-We have the following deliverables on our intermediate-term roadmap (not in order of completion):
-
-* [x] Support General-purpose input/output protocols: GPIO, SPI, I2C, and PWM
-* [ ] Support common serial protocols: RS-485 Serial Port, CAN bus, Modbus
-* [ ] Support digital audio bus protocols: I2S
-* [x] Support Raspberry Pi 3 on Linux and Windows 10 IoT Core RS5
-* [ ] Support Hummingboard Edge on Linux and Windows 10 IoT Core RS5
-* [ ] Support BeagleBoard Black on Linux and Windows 10 IoT Core RS5
-* [x] Support sysfs (/sys/class/gpio) for Generic/Portable Device Driver on Linux Kernel 3.6+
-* [ ] Support libgpiod (/dev/gpiochipX) for Generic/Portable Device Driver on Linux Kernel 4.8+
-* [ ] Stabilize System.Device.Gpio API
-* [x] Publish [System.Device.Gpio to NuGet.org](https://www.nuget.org/packages/System.Device.Gpio)
-* [x] Provide dockerfiles for all samples
-* [ ] Publish Docker images for a subset of samples
-* [ ] Support x64
-* [x] Support ARM32
-* [ ] Support ARM64
-* [ ] Support Device Bindings for common sensors and microcontrollers such as those bundled with the [Microsoft IoT Starter Pack](https://www.adafruit.com/product/2733)
-* [ ] Support Device Bindings and Canvas Widgets for LCDS, multi-panel [LED matrices](https://www.adafruit.com/product/607), and Displays
-* [ ] Support Device Bindings for servo and stepper motors, motor controllers, and drives
-* [ ] Stabilize IoT.Device.Bindings APIs for Device Bindings
-* [x] Publish [IoT.Device.Bindings to NuGet.org](https://www.nuget.org/packages/Iot.Device.Bindings)
+While this repo is currently in preview, we want to focus our efforts in a particular direction, specifically to work on areas aligned with our [roadmap](Documentation/roadmap.md).
```

Comment:
I'd collapse this to a just a link to Roadmap

Reply:
Will move to the Getting Started section.